insights/client/collection_rules.py

```diff
@@ -99,20 +100,25 @@ def get_collection_rules(self, raw=False):
                      self.collection_rules_url)
 
         net_logger.info("GET %s", self.collection_rules_url)
-        req = self.conn.session.get(
-            self.collection_rules_url, headers=({'accept': 'text/plain'}))
+        try:
+            req = self.conn.session.get(
+                self.collection_rules_url, headers=({'accept': 'text/plain'}))
 
-        if req.status_code == 200:
-            logger.debug("Successfully downloaded collection rules")
+            if req.status_code == 200:
+                logger.debug("Successfully downloaded collection rules")
 
-            json_response = NamedTemporaryFile()
-            json_response.write(req.text)
-            json_response.file.flush()
-        else:
-            logger.error("ERROR: Could not download dynamic configuration")
-            logger.error("Debug Info: \nConf status: %s", req.status_code)
-            logger.error("Debug Info: \nConf message: %s", req.text)
-            return False
+                json_response = NamedTemporaryFile()
+                json_response.write(req.text)
+                json_response.file.flush()
+            else:
+                logger.error("ERROR: Could not download dynamic configuration")
+                logger.error("Debug Info: \nConf status: %s", e.status_code)
+                logger.error("Debug Info: \nConf message: %s", e.text)
```

Comment:
On these two debug lines, did you mean `req.status_code` and `req.text` instead of `e`?

Reply:
Yup, small typo there. Will fix

Reply:
fixed~